nucleus/payara-modules/hazelcast-bootstrap/src/main/java/fish/payara/nucleus/eventbus/EventBus.java

```diff
@@ -68,13 +69,13 @@
     @Inject
     private Events events;
     
-    private HashMap<String,TopicListener> messageReceivers;
+    private ConcurrentHashMap<String,TopicListener> messageReceivers;
```

Comment:
This can probably just be a Map (the rest of the code has no need for the detail that it will later be assigned a concurrent map)